From e067ece6bddc3ac3db5864e2c55061142794fa6c Mon Sep 17 00:00:00 2001
From: Gergely Nagy <algernon@madhouse-project.org>
Date: Tue, 18 Mar 2014 12:31:48 +0100
Subject: [PATCH] aalinuxkbd: Always return with a value

Since rawmode_init() needs to return an int, make sure that we return
with a value, even when we bail out early. This fixes Debian #739932.

Reported-by: Arthur Marble <arthur@info9.net>
Signed-off-by: Gergely Nagy <algernon@madhouse-project.org>
---
 src/aalinuxkbd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/aalinuxkbd.c b/src/aalinuxkbd.c
index 610e026..918cb3a 100644
--- a/src/aalinuxkbd.c
+++ b/src/aalinuxkbd.c
@@ -200,7 +200,7 @@ static void vt_to_here(int num)
 static int rawmode_init(void)
 {
     if (!closed)
-	return;
+       return (1);
     mypid = getpid();
     if (tty_fd == -1) {
 	tty_fd = fileno(stdin);
